topics/assembly/tutorials/get-started-genome-assembly/slides.html

```diff
@@ -15,21 +17,109 @@
   - r1corre
   funding:
   - erasmusplus
-level: Introductory
+level: Introductory 
+
+---
+
+# The truth about bioinformatics
+
+.image-100[
+  ![Bioinformatics is not just about pushing a button and getting your result](../../images/genome_assembly_introduction/the_truth_about_bioinformatics.jpg)
+]
+
+---
+
+# Let's start with some important definitions
+
+.image-40[
+  ![Importance to speak the same language](../../images/genome_assembly_introduction/bioinformatics_terms_def.jpg)
```

Comment:
Not sure if we have the right to copy a Gaston image :D

Reply:
:(